### ISSUE TITLE
Proposal to rename ele-testhelpers to testhelpers

### ISSUE THREAD

**Issue by valaparthvi** [rancher-sandbox/ele-testhelpers#63]
This repo is used by more than just the elemental team, r/hosted-providers-e2e, r/fleet-e2e, r/<gke/aks/eks>-operator to name a few among [others](https://github.com/rancher-sandbox/ele-testhelpers/network/dependents), so I propose to rename this repository to a more generic name, **_testhelpers_** or **_rancher-testhelpers_** for e.g.

**Comment by ldevulder** [comment]
Yes already thought of this. I will try to prepare something within the next 2 weeks.
